### ISSUE TITLE
Events from subscription not displayed correctly on "2 weekly" and "monthly" views

### ISSUE THREAD

**Issue by Xemnas0** [haydenull/logseq-plugin-agenda#55]
Events from calendar subscriptions are not displayed correctly in monthly and 2 weekly view, but correctly in weekly and daily view.
This issue makes reading the event titles extremely difficult.

Daily (Ok)
![image](https://user-images.githubusercontent.com/26822543/170862382-73edc936-ffbc-4c21-8dda-45f724aca2e9.png)

Weekly (Ok)
![image](https://user-images.githubusercontent.com/26822543/170862423-4733c9ac-c83e-47fb-a3ba-f10eeffb1f99.png)

2 weekly (NOT ok)
![image](https://user-images.githubusercontent.com/26822543/170862462-e0af73f0-e833-4784-8b21-e78e00429c34.png)

Monthly (NOT ok)
![image](https://user-images.githubusercontent.com/26822543/170862489-e0fc06c3-5c7d-4304-b239-116793da67b8.png)

**Comment by haydenull** [comment]
Are you referring to events that don't appear in the timeline?

**Comment by Xemnas0** [comment]
No. Events appear, but they do not have the background and text color that they should have, making them impossible to read.
According to my settings, they should have green background and red text, which only happens in Daily and Weekly view

**Comment by haydenull** [comment]
Events are divided into all-day and non-all-day, non-all-day events have no background color, is it related to this？

**Comment by milanb** [comment]
I have same problem. It looks like this:

<img width="450" alt="image" src="https://user-images.githubusercontent.com/435881/174410626-887574bf-60f4-4060-bd0c-c75d1e674b2b.png">

**Comment by haydenull** [comment]
Version 2.4.2 should fix this, you can try it again.

**Comment by milanb** [comment]
Now it works. Thank You!

<img width="125" alt="image" src="https://user-images.githubusercontent.com/435881/174792216-48c57eeb-cfd8-45e5-9c8b-2d5c48bbfd62.png">
